lib/ui/src/modules/ui/components/left_panel/index.js

```diff
@@ -15,7 +15,13 @@ const mainStyle = {
   padding: '10px 0 10px 10px',
 };
 
-const storyProps = ['stories', 'selectedKind', 'selectedStory', 'onSelectStory'];
+const storyProps = [
+  'storiesHierarchy',
+  'selectedKind',
+  'selectedHierarchy',
+  'selectedStory',
```

Comment:
I think linter cries that they are not in use in here. these props just passed further to stories.js where they are actually defined in prop types.